test/integration/targets/become/aliases

```diff
@@ -2,3 +2,4 @@ destructive
 shippable/posix/group1
 context/target
 gather_facts/no
+setup/always/setup_passlib_controller
```

Comment:
@mattclay I added this for every target that uses `setup_test_user`. I feel like `setup_passlib_controller` should be run as a dependency/part of `setup_test_user` but I couldn't make it work for `setup_passlib_controller` to be run always on a controller side as `setup_test_user` is used either as a meta dependency of `targets` or as `needs/target/setup_test_user` and called explicitly in a test either in `roles:` or via `include_role`.

Reply:
Yeah, currently `setup/always/` doesn't work recursively. It's detected as a dependency, but it's not actually invoked.

Reply:
I added a note that it's required by `setup_test_user`, at least.